src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -105,7 +105,13 @@ namespace AppInstaller::CLI::Workflow
         std::shared_ptr<Repository::ISource> source;
         try
         {
-            source = context.Reporter.ExecuteWithProgress(std::bind(Repository::OpenSource, sourceName, std::placeholders::_1), true);
+            auto result = context.Reporter.ExecuteWithProgress(std::bind(Repository::OpenSource, sourceName, std::placeholders::_1), true);
+            source = result.Source;
+
+            for (const auto& s : result.SourcesWithUpdateFailure)

```

Comment:
>for [](start = 12, length = 3)

add a comment indicating that this is only going to produce a warning but we'll continue with the operation (install/update etc.)